media/video/videobuf-dma-contig.c fails to compile on ARM Versatile
like this:
 | videobuf-dma-contig.c: In function ‘videobuf_dma_contig_user_get’:
 | videobuf-dma-contig.c:139: error: dereferencing pointer to incomplete type
 | videobuf-dma-contig.c:184: error: dereferencing pointer to incomplete type
 | make[8]: *** [drivers/media/video/videobuf-dma-contig.o] Error 1

Looking at the preprocessed source, I noticed that there was no definition
for struct task_struct.

Signed-off-by: Martin Michlmayr <tbm@cyrius.com>

--- a/drivers/media/video/videobuf-dma-contig.c	2009-10-31 10:22:42.000000000 +0000
+++ b/drivers/media/video/videobuf-dma-contig.c	2009-10-31 10:24:40.000000000 +0000
@@ -19,6 +19,7 @@
 #include <linux/mm.h>
 #include <linux/pagemap.h>
 #include <linux/dma-mapping.h>
+#include <linux/sched.h>
 #include <media/videobuf-dma-contig.h>
 
 struct videobuf_dma_contig_memory {

